vite.config.ts

```diff
@@ -51,11 +51,7 @@ export default defineConfig({
   },
   plugins: [
     react(),
-    svgr({
-      svgrOptions: {
-        icon: true,
-      },
-    }),
+    svgr(),
```

Comment:
Noted on this - probably worth just validating when it's been deployed that the SVG behaviour hasn't broken